lib/msal-core/test/ServerRequestParameters.spec.ts

```diff
@@ -128,4 +128,22 @@ describe("ServerRequestParameters.ts Class", function () {
             expect(err.errorMessage).to.include(ClientConfigurationErrorMessage.invalidCorrelationIdError.desc);
         });
     });
+
+    describe("Query Parameters", function () {
+
+        it("test hints populated using queryParameters", function () {
+            const eQParams = {domain_hint: "Coke.com", locale: "en-us"};
+            const extraQueryParameters = ServerRequestParameters.generateQueryParametersString(eQParams);
+            expect(extraQueryParameters).to.include("domain_hint");
+            expect(extraQueryParameters).to.include("locale");
+        });
+
+        it("test hints populated using queryParameters", function () {
+            const eQParams = {domain_hint: "Coke.com", locale: "en-us"};
```

Comment:
Not sure how the `coke.com` folks would feel but might be better to use `microsoft.com` as a test case.

Reply:
:-D I think "Coke.com" raised this issue and playing in my head. Will change it.

Reply:
Thank you for taking this on @sameerag . Any idea when it will be published?

Reply:
We stalled the release as we want this a part of 1.3.x release. We now released 1.2.2, so this should be soon merged to dev and released.

Reply:
Addressed "Coke.com" comment